e2e/test/FaultInjection.cs

```diff
@@ -185,7 +186,7 @@ public static async Task ActivateFaultInjection(Client.TransportType transport,
                     bool isFaulted = false;
                     for (int i = 0; i < LatencyTimeBufferInSec; i++)
                     {
-                        if (connectionStatusChangeCount >= 2)
+                        if (connectionStatusChangeCount > countBeforeFaultInjection)
```

Comment:
Can you elaborate why is this changed to countBeforeFaultInjection which is set to connectionStatusChangeCount ?